wiki/mnmg/Using_Infiniband_for_MNMG.md

```diff
@@ -0,0 +1,332 @@
+# Using Infiniband for Multi-Node Multi-GPU cuML
+
+These instructions outline how to run multi-node multi-GPU cuML on devices with Infiniband. These instructions assume the necessary Infiniband hardware has already been installed and the relevant software has already been configured to enable communication over the Infiniband devices.
+
+## 1. Install UCX 
+
+### From Conda
+
+Note: this package currently requires CUDA9.2. A CUDA10 package is in the works.
+
+`conda install -c conda-forge -c jakirkham/label/ucx cudatoolkit=9.2 ucx-proc=*=gpu ucx python=3.7`
+
+### From Source
+
+Install autogen if it's not already installed:
+```bash
+sudo apt-get install autogen autoconf libtool
+```
+
+Optionaly install `gdrcopy` for faster GPU-Network card data transfer: 
+
+From the [ucx wiki](https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support), `gdrcopy` can be installed, and might be necessary, to enable faster GPU-Network card data transfer.
+
+Here are the install instructions, taken from [gdrcopy github](https://github.com/NVIDIA/gdrcopy)
+```bash
+git clone https://github.com/NVIDIA/gdrcopy.git
+cd gdrcopy
+make PREFIX=$CONDA_INSTALL_PREFIX CUDA=/usr/local/cuda
+sudo ./insmod.sh
+export LD_LIBRARY_PATH=$PWD:$LD_LIBRARY_PATH
```

Comment:
If we install this to the conda library path, this may not be necessary.

Reply:
This is true. This can be removed since I’m installing directly to conda.